hack/make/tools.mk

```diff
@@ -57,7 +57,7 @@ endif
 endif
 
 GOLANGCI_URL := https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh
-GOLANGCI_VERSION := 1.49.0
+GOLANGCI_VERSION := 1.53.3
```

Comment:
Should it be `v1.53.3`?  I keep getting validation error if I drop `v` .

Reply:
https://github.com/golangci/golangci-lint/releases/tag/v1.53.3

I have the above latest version